planner/core/util.go

```diff
@@ -231,3 +235,33 @@ func GetStatsInfo(i interface{}) map[string]uint64 {
 	statsInfos = CollectPlanStatsVersion(physicalPlan, statsInfos)
 	return statsInfos
 }
+
+// StringList used to sort the string list
+type StringList []string
+
+// Len implement sort.Interface
+func (s StringList) Len() int { return len(s) }
+
+// Swap implement sort.Interface
+func (s StringList) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+
+// Less implement sort.Interface
+func (s StringList) Less(i, j int) bool { return s[i] < s[j] }
+
+// extractStringFromStringSet helps extract string info from set.StringSet
+func extractStringFromStringSet(set set.StringSet) string {
+	r := new(bytes.Buffer)
+	if len(set) < 1 {
+		return r.String()
+	}
+	l := make([]string, 0, len(set))
+	for k := range set {
+		l = append(l, k)
+	}
+	sl := StringList(l)
+	sort.Sort(sl)
```

Comment:
You can use `	sort.Strings()`, and remove line #238 ~ 250.

Reply:
updated.